report/include/scp/report.h

```diff
@@ -55,6 +74,28 @@ enum class log {
     TRACEALL,
     DBGTRACE = TRACEALL
 };
+
+static const std::array<sc_core::sc_severity, 8> severity = {
```

Comment:
This leads to code bloat as **any** translation unit instantiates this array. Declare it extern and  change the definition in report.cpp by removing the static modifier